sos/plugins/ovn_host.py

```diff
@@ -35,7 +35,7 @@ def setup(self):
         self.add_cmd_output([
             'ovs-ofctl -O OpenFlow13 dump-flows br-int',
             'ovs-vsctl list-br',
-            'ovs-vsctl list OpenVswitch',
+            'ovs-vsctl list Open_vSwitch',
```

Comment:
My only question is, was OpenVswitch ever a correct value here, in any prior versions of OVS? It's been in there since the plugin was first added, so I'd be surprised if we've gone all this time without it being reported.

Reply:
I think it's been unnoticed since the beginning... If you check OVS 1.1 (year 2011) [0] it was Open_vSwitch.

[0] https://github.com/openvswitch/ovs/blob/branch-1.1/vswitchd/vswitch.ovsschema#L5